src/main/java/org/broadinstitute/hellbender/tools/walkers/coverage/DepthOfCoverage.java

```diff
@@ -345,6 +345,7 @@ public void onIntervalEnd(final Locatable activeInterval) {
                 for (DoCOutputType.Partition p : partitionTypes) {
                     // Write the per-interval depth information as necessary
                     final DepthOfCoverageStats coverageByAggregationPartitionType = partitionerToRemove.getCoverageByAggregationType(p);
+                    writer.writePerIntervalDepthInformation(p, (SimpleInterval) activeInterval, coverageByAggregationPartitionType, globalIdentifierMap.get(p));
```

Comment:
This seems like a fairly significant bug... did no one ever look at the output file? or are there multiple?

Reply:
There are 25 output files for each of these test files, and the one that was checked in was generated correctly before the rebase in question so it was missed. This only affects 4 of the 25 output files from this tool.